script/update

```diff
@@ -237,7 +237,7 @@ def gn_args(target_arch, component, gyp_defines):
   if component == 'shared_library':
     args.append('is_component_build=true')
   else:
-    args.append('is_component_build=false is_official_build=true')
+    args.append('is_component_build=false')
```

Comment:
It would generate `LNK1107` error when generating `libv8_base.lib`, by removing the `/GL` linker parameter which is added by `is_official_build=true`, the error is gone.

Chromium does not have this error because v8 is a `source_set` in it.